src/main/java/edu/harvard/iq/dataverse/DashboardPage.java

```diff
@@ -189,5 +191,19 @@ public boolean isSuperUser() {
         return session.getUser().isSuperuser();
     }
 
+    /**
+     * Calculates and returns count of active and inactive licenses in the whole Dataverse.
+     *
+     * @return active and inactive licenses count
+     */
+    public Tuple2<Long, Long> getActiveAndInactiveLicensesCount() {
+        List<License> licenses = licenseDAO.findAll();
+
+        long activeLicenses = licenses.stream().filter(License::isActive).count();
+        long inactiveLicenses = licenses.size() - activeLicenses;
```

Comment:
I would extend dto functionality to get counts - no need to fetch full objects

Reply:
Where would I get the dto from :P? The licensesDto that we are operation on is on another page which is uninitalized here.